src/runner/main.cpp

```diff
@@ -22,6 +23,17 @@
 
 extern "C" IMAGE_DOS_HEADER __ImageBase;
 
+namespace localized_strings
+{
+    const wchar_t MSI_VERSION_IS_ALSO_RUNNING[] = L"Warning: MSI version is also running!";

```

Comment:
If the MSI version is running, this warning is shown before the warning that offers to uninstall the MSI?

Reply:
at the same time. It's a toast.

Reply:
We should call `start_msi_uninstallation_sequence` before showing other warnings.

If the user doesn't uninstall or doesn't quit the MSI version, the MSIX runner should exit with an error dialog saying:
"An older version of PowerToys is already running."